force-app/main/default/triggers/OpportunityTrigger.trigger

```diff
@@ -0,0 +1,44 @@
+trigger OpportunityTrigger on Opportunity (before update, before delete) {
+    if (Trigger.isBefore && Trigger.isUpdate){
+        Set<Id> accountIds = new Set<Id>();
+        for(Opportunity opp : Trigger.New){
+            if (opp.AccountId != null) {
+                accountIds.add(opp.AccountId);
+            }
+            if(opp.Amount <= 5000 && opp.Amount != null){
+                opp.addError('Opportunity amount must be greater than 5000');
+            }
+        }
+        Map<Id, Contact> accountToCeoContact = new Map<Id, Contact>();
```

Comment:
Try to separate this into a different if Statement so that it can be tested separately.